apm-sniffer/apm-sdk-plugin/thrift-plugin/src/main/java/org/apache/skywalking/apm/plugin/thrift/wrapper/ServerInProtocolWrapper.java

```diff
@@ -57,10 +64,11 @@ public void initial(AbstractContext context) {
     @Override
     public TField readFieldBegin() throws TException {
         final TField field = super.readFieldBegin();
+        Map<String, String> header = new HashMap<>(1);
```

Comment:
The variable is visible in the if statement block. So we don't need to move it out of the if statement block.

Reply:
done